### PR TITLE
fix(all): fix ESLint errors

### PR DESCRIPTION
**Pull request by carlos-ds**
Fixed all linting errors that were returned when running `yarn run lint:all`.

```
  yarn run lint:all --skip-nx-cache
yarn run v1.22.22
$ nx run-many --all --target=lint --skip-nx-cache

 NX   --skip-nx-cache disables the connection to Nx Cloud for the current run.

The remote cache will not be read from or written to during this run.


   ✔  nx run mini-rx-store:lint (4s)
   ✔  nx run common:lint (5s)
   ✔  nx run mini-rx-store-ng:lint (6s)
   ✔  nx run signal-store:lint (2s)
   ✔  nx run signal-store-angular-demo-standalone:lint (2s)
   ✔  nx run signal-store-angular-demo-standalone-e2e:lint (798ms)
   ✔  nx run mini-rx-angular-demo-e2e:lint (766ms)
   ✔  nx run mini-rx-angular-demo:lint (2s)

—————————————————————————————————————————————————————————————————————————————————————————————————————————————

 NX   Successfully ran target lint for 8 projects (7s)

✨  Done in 8.67s.
```

### CODE REVIEW

File: apps/signal-store-angular-demo-standalone/src/app/app.component.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { UserFacade } from './user/state/user-facade.service';
 @Component({
     standalone: true,
     imports: [RouterModule],
-    selector: 'mini-rx-root',
+    selector: 'app-mini-rx-root',
```
Comment (spierala) [inline apps/signal-store-angular-demo-standalone/src/app/app.component.ts:9]: app-root

File: apps/signal-store-angular-demo-standalone/src/index.html [hunk 2]
```diff
@@ -8,6 +8,6 @@
         <link rel="icon" type="image/x-icon" href="favicon.ico" />
     </head>
     <body>
-        <mini-rx-root></mini-rx-root>
+        <app-mini-rx-root></app-mini-rx-root>
```
Comment (spierala) [inline apps/signal-store-angular-demo-standalone/src/index.html:11]: app-root

File: libs/common/src/lib/deep-freeze.spec.ts [hunk 3]
```diff
@@ -62,7 +62,9 @@ describe('deepFreeze', () => {
             str: 'immutable',
             num: 42,
             bool: true,
-            fn: function () {},
+            fn: function () {
```
Comment (spierala) [inline libs/common/src/lib/deep-freeze.spec.ts:65]: minor: maybe use fat arrow fn?

fn: () => {}

File: libs/mini-rx-store/src/lib/selector.ts [hunk 4]
```diff
@@ -114,7 +114,7 @@ export function createSelector(...args: any[]): Selector<any, any> {
 
     return memoizeOne((state) => {
         const selectorResults = selectors.map((fn) => fn(state));
-        return memoizedProjector.apply(null, selectorResults);
+        return memoizedProjector(...selectorResults);
```
Comment (spierala) [inline libs/mini-rx-store/src/lib/selector.ts:117]: better indeed! 👍

File: libs/common/src/lib/deep-freeze.spec.ts (hunk 3 above)
Comment (carlos-ds) [reply libs/common/src/lib/deep-freeze.spec.ts:65]: The arrow function is not an issue but with your suggestion, linting will still error on `@typescript-eslint/ no-empty-function` so I will keep the explicit `return`

**Pull request merged**